### PR TITLE
Distinguised Footer

### PR DESCRIPTION
**Pull request by darkdrgn2k**
Darken footer, reorganize some text and center part of #154

### CODE REVIEW

**Comment by darkdrgn2k** [comment]
Just an idea to tie of some of the loose ends in the ISSUES column. Thoughts, Ideas, Comments, Concerns, other things?

File: _includes/footer.html [hunk 1]
```diff
@@ -19,7 +19,8 @@
       </div>
       <!-- copyright info -->
       <div class="twelve columns site-footer-cc">
-        <p> &copy; {{ 'now' | date: "%Y" }}, Toronto Mesh. Our <a href="/code-of-conduct/">Code of Conduct</a>.<br />This website content is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International</a> license.</p>
+        <p> &copy; {{ 'now' | date: "%Y" }}, Toronto Mesh.<br />This website content is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International</a> license.</p>
+        <p><a href="/code-of-conduct/">Code of Conduct</a></p>
```
Comment (dcwalk) [inline _includes/footer.html]: add "." at end.

Also, could we keep these elements closer together? 
I'm not tied to the ordering. But I think having "Code of Conduct? as a dangling paragraph feels unbalanced.

File: _sass/_footer.scss [hunk 2]
```diff
@@ -1,8 +1,10 @@
 // Site footer
 
 .site-footer {
-  margin: 3em 0;
```
Comment (dcwalk) [inline _sass/_footer.scss:4]: I think we might want more spacing on top and bottom if we have a different background. I'm looking through the secondary pages (e.g., events) and it the spacing feels off
screen shot 2017-09-11 at 12 18 53 pm

File: _includes/footer.html (hunk 1 above)
Comment (darkdrgn2k) [reply _includes/footer.html]: how bout removing below  since its in submenu now?

File: _sass/_footer.scss (hunk 2 above)
Comment (darkdrgn2k) [reply _sass/_footer.scss:4]: good idea, it was looking wierd with the margin, but padding looks better.

File: _includes/footer.html (hunk 1 above)
Comment (darkdrgn2k) [reply _includes/footer.html]: (unless we want to add like a privacy policy or terms or something then we can list them all there too)

Comment (garrying) [reply _includes/footer.html]: Code of Conduct feels a bit off by itself. Maybe we can incorporate into a sentence?

File: _sass/_footer.scss (hunk 2 above)
Comment (garrying) [reply _sass/_footer.scss:4]: Minor adjustments in 029191b

**Comment by dcwalk** [comment]
I think this looks good, tho see my comment in #173 for potential future shifts to be in sync with post-refresh site-- the CI failure is from the makerfestival website, which I've confirmed separately is having trouble loading atm.

**Review by dcwalk — APPROVED** [review]

**Pull request merged**